consensus-types/interfaces/beacon_block.go

```diff
@@ -77,6 +77,7 @@ type ReadOnlyBeaconBlockBody interface {
 	Execution() (ExecutionData, error)
 	BLSToExecutionChanges() ([]*ethpb.SignedBLSToExecutionChange, error)
 	BlobKzgCommitments() ([][]byte, error)
+	Consolidations() ([]*ethpb.SignedConsolidation, error)
```

Comment:
I added set consolidations in my branch is it not needed at all?

Reply:
It might be needed, but not at this time. This PR is only adding things that were found when trying to pass spec tests